packages/firebase_ml_vision/lib/src/image_labeler.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
This file was renamed from `label_detector.dart` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).